build.sbt

```diff
@@ -21,32 +21,32 @@ asciiGraphWidth := 999999999 // to ensure Snyk can read the the deeeeep dependen
 libraryDependencies ++= Seq(
   "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
   "com.gu.play-googleauth" %% "play-v30" % "19.0.0",
-  "com.google.cloud" % "google-cloud-bigquery" % "2.43.1",
+  "com.google.cloud" % "google-cloud-bigquery" % "2.43.3",
   "com.gu" %% "simple-configuration-ssm" % "1.5.8",
   "software.amazon.awssdk" % "s3" % awsVersion,
   "software.amazon.awssdk" % "dynamodb" % awsVersion,
   "io.circe" %% "circe-core" % circeVersion,
   "io.circe" %% "circe-generic" % circeVersion,
   "io.circe" %% "circe-parser" % circeVersion,
-  "io.circe" %% "circe-generic-extras" % circeVersion,
+  "io.circe" %% "circe-generic-extras" % "0.14.4",
```

Comment:
sadly the versioning of circe-generic-extras has diverged from the rest of circe